src/test/resources/test-users.json

```diff
@@ -0,0 +1,99 @@
+{
+  "header" : {
+  "alg": "RS256",
+  "typ": "JWT",
+  "kid": "XbBBJU9vyCmVDxPjWZfcEy70ZIQkZK6xV2lxnOOpqUk"
+},
+  "write_user":
+        {"jti": "c81dc116-6c85-440c-96db-2655785754f7",
+        "exp": 1571730938,
+        "nbf": 0,
+        "iat": 1571730638,
+        "iss": "https://sso.ut1.fellesdatakatalog.brreg.no/auth/realms/fdk",
+        "aud": [
+          "a-backend-service",
```

Comment:
Hva med å ha et token som mangler audience for a-backend-service i tillegg?

Reply:
Er ikke det noe Spring Secuirty tar seg av?

Reply:
All sikkerhet må vere spekka i spekken. (Sjølv om det ikkje har konsekvenser for kodegenerering. Spekken er viktig for den eller det som skal bruke API-et.) Vi kan sjå på det saman.